src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/payment/processor/helper/PaymentsMessageSender.java

```diff
@@ -0,0 +1,64 @@
+package uk.gov.hmcts.reform.bulkscan.payment.processor.helper;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.microsoft.azure.servicebus.IMessage;
+import com.microsoft.azure.servicebus.Message;
+import com.microsoft.azure.servicebus.QueueClient;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.context.annotation.Import;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.bulkscan.payment.processor.FunctionalQueueConfig;
+import uk.gov.hmcts.reform.bulkscan.payment.processor.model.CreatePaymentsCommand;
+
+import java.time.Instant;
+import java.util.UUID;
+
+import static org.springframework.http.MediaType.APPLICATION_JSON;
+
+@Service
+@Import(FunctionalQueueConfig.class)
+@Profile("nosb")
+public class PaymentsMessageSender {
+
+    private static final Logger LOG = LoggerFactory.getLogger(PaymentsMessageSender.class);
+
+    private static final String CREATE = "CREATE";
+
+    @Autowired
+    @Qualifier("payments")
+    private QueueClient queueClient;
+
+    private final ObjectMapper objectMapper = new ObjectMapper();
+
+    public void send(CreatePaymentsCommand cmd) {
+        try {
+            final String messageContent = objectMapper.writeValueAsString(cmd);
+
+            IMessage message = new Message(
+                "test_msg_" + UUID.randomUUID().toString(),
+                messageContent,
+                APPLICATION_JSON.toString()
+            );
+            message.setLabel(CREATE);
+
+            long res = queueClient.scheduleMessage(message, Instant.now());
```

Comment:
doesn't say much. what is this result number?

Reply:
renamed to sentMessageSequenceNumber